src/app/domain/backup/services/manageBackup.ts

```diff
@@ -0,0 +1,124 @@
+import Minilog from '@cozy/minilog'
+
+import { managePermissions } from '/app/domain/backup/services/managePermissions'
+import {
+  getLocalBackupConfig,
+  initiazeLocalBackupConfig,
+  setBackupAsInitializing,
+  setBackupAsReady,
+  setBackupAsRunning,
+  setBackupAsDone
+} from '/app/domain/backup/services/manageLocalBackupConfig'
+import { getMediasToBackup } from '/app/domain/backup/services/getMedias'
+import { uploadMedias } from '/app/domain/backup/services/uploadMedias'
+import {
+  fetchDeviceRemoteBackupConfig,
+  fetchBackupedMedias,
+  createRemoteBackupFolder
+} from '/app/domain/backup/services/manageRemoteBackupConfig'
+import {
+  activateKeepAwake,
+  deactivateKeepAwake
+} from '/app/domain/sleep/services/sleep'
+
+import type CozyClient from 'cozy-client'
+
+import {
+  BackupInfo,
+  BackupedMedia,
+  LocalBackupConfig
+} from '/app/domain/backup/models'
+
+const log = Minilog('💿 Backup')
+
+export const prepareBackup = async (
+  client: CozyClient
+): Promise<BackupInfo> => {
+  log.debug('Backup preparation started')
+
+  await managePermissions()
+
+  const backupConfig = await initializeBackup(client)
+
+  if (backupConfig.currentBackup.status === 'running') {
```

Comment:
initializeBackup returns no `currentBackup` if the method throws and creates a new backupconfig, whill this throw?